tests/apps/slurm/test_api.py

```diff
@@ -1,4 +1,4 @@
-from typing import Dict, List, Optional
+from typing import Optional
```

Comment:
I think we can include in this PR changing the occurrences of `Optional[type]` for `type | None`

Reply:
there is one occurrence in this file in line 137

Reply:
Let's do that PEP in another PR.